src/main/java/com/shippo/ExamplePickup.java

```diff
@@ -0,0 +1,147 @@
+		// don't have one? get more info here (https://goshippo.com/docs/#overview)
+		Shippo.setApiKey("shippo_test_cf1b6d0655e59fc6316880580765066038ef20d8");
+		Shippo.setApiVersion("2018-02-08");
+
+		// Optional defaults to false
+		//Shippo.setDEBUG(true);
+
+		// to address
+		Map<String, Object> toAddressMap = new HashMap<String, Object>();
+		toAddressMap.put("name", "Mr Hippo");
+		toAddressMap.put("company", "Regents Park");
+		toAddressMap.put("street1", "Outer Cir");
+		toAddressMap.put("city", "London");
+		toAddressMap.put("state", null);
+		toAddressMap.put("zip", "NW1 4RY");
+		toAddressMap.put("country", "GB");
+		toAddressMap.put("phone", "+1 555 341 9393");
+		toAddressMap.put("email", "ms-hippo@goshipppo.com");
+		toAddressMap.put("is_residential", false);
+		toAddressMap.put("metadata", "For Order Number 123");
+
+		// from address
+		Map<String, Object> fromAddressMap = new HashMap<String, Object>();
+		fromAddressMap.put("name", "Ms Hippo");
+		fromAddressMap.put("company", "San Diego Zoo");
+		fromAddressMap.put("street1", "2920 Zoo Drive");
+		fromAddressMap.put("city", "San Diego");
+		fromAddressMap.put("state", "CA");
+		fromAddressMap.put("zip", "92101");
+		fromAddressMap.put("country", "US");
+		fromAddressMap.put("email", "mshippo@goshipppo.com");
+		fromAddressMap.put("phone", "+1 619 231 1515");
+		fromAddressMap.put("metadata", "Customer ID 123456");
+
+		// parcel
+		Map<String, Object> parcelMap = new HashMap<String, Object>();
+		parcelMap.put("length", "5");
+		parcelMap.put("width", "5");
+		parcelMap.put("height", "5");
+		parcelMap.put("distance_unit", "in");
+		parcelMap.put("weight", "2");
+		parcelMap.put("mass_unit", "lb");
+		List<Map<String, Object>> parcels = new ArrayList<Map<String, Object>>();
+		parcels.add(parcelMap);
+
+		// customs item
+		Map<String, Object> customsItemMap = new HashMap<String, Object>();
+		customsItemMap.put("description", "T-Shirt");
+		customsItemMap.put("quantity", "2");
+		customsItemMap.put("net_weight", "400");
+		customsItemMap.put("mass_unit", "g");
+		customsItemMap.put("value_amount", "20");
+		customsItemMap.put("value_currency", "USD");
+		customsItemMap.put("origin_country", "US");
+		customsItemMap.put("tariff_number", "");
+		List<Map<String, Object>> customsItem = new ArrayList<Map<String, Object>>();
+		customsItem.add(customsItemMap);
+
+		// customs declaration
+		Map<String, Object> customsDeclarationMap = new HashMap<String, Object>();
+		customsDeclarationMap.put("contents_type", "MERCHANDISE");
+		customsDeclarationMap.put("contents_explanation", "T-Shirt purchase");
+		customsDeclarationMap.put("non_delivery_option", "RETURN");
+		customsDeclarationMap.put("certify", "true");
+		customsDeclarationMap.put("certify_signer", "Mr Hippo");
+		customsDeclarationMap.put("items", customsItem);
+		System.out.println("Creating customs declaration...");
+        CustomsDeclaration customsDeclaration = CustomsDeclaration.create(customsDeclarationMap);
```

Comment:
some spacing weirdness in this file as well

Reply:
Spaces & Tabs used in different files, I've installed a linter to fix this